spec/feature/user/edits_a_listing_spec.rb

```diff
@@ -14,7 +14,7 @@
     expect(page).to have_content("Edit listing")
   end
 
-  it "and is redirected to the edited listing and sees the edits" do
+  xit "and is redirected to the edited listing and sees the edits" do
```

Comment:
This is behaviour needs to tested.
Capybara will see the dom elements in the drop-down.  The test just needs to be tweaked a little.